prisma/schema.prisma

```diff
@@ -15,25 +15,25 @@ model User {
   createdAt   DateTime      @default(now())
   updatedAt   DateTime      @updatedAt
   deactivated Boolean       @default(false)
-  record      Record[]
+  dnsRecord   DnsRecord[]
   certificate Certificate[]
 }
 
-model Record {
-  id           Int          @id @default(autoincrement())
+model DnsRecord {
```

Comment:
When this is merged, ping me to figure out how we'll migrate the staging db.

cc @dadolhay.